scipy/stats/stats.py

```diff
@@ -3136,6 +3148,13 @@ def f_oneway(*args):
     pvalue : float
         The associated p-value from the F-distribution.
 
+    Warns 
+    -------
+    F_onewayConstantInputWarning
+        Raised if each of the input arrays is constant array.
+        In this case F-value should be either infinite or isn't defined, so
```

Comment:
here maybe state: 
    In this case the F-statistic is either infinite or not defined, so ...

Reply:
Did, but left "F-value"